cli/package.json

```diff
@@ -14,7 +14,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@epfml/disco-server": "*",
+    "server": "*",
```

Comment:
Why not use the npm server anymore?

Reply:
the server is not published to NPM, so no need to have it prefixed by the organisation